ci/setup_env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I would prefer not to add an entire file for this kind of thing. Plus, this file only works on some operating systems.